h/static/scripts/login-forms/components/AccountSettingsForms.tsx

```diff
@@ -178,6 +178,8 @@ function ConnectAccountButton({
           <span className="grow">
             Connected:{' '}
             <a
+              target="_blank"
+              rel="noreferrer"
```

Comment:
The lint warning that asks you to add `rel="noreferrer"` when setting `target="_blank"` is obsolete now and we should change it. noreferrer here has the mildly undesirable effect of hiding from ORCID that the link came from https://hypothes.is.

The historical reason was to prevent the popup (or new tab) from being able to navigate the window that opened it via `window.opener`.

For this PR, merge as-is since the downside is very minor.